src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs

```diff
@@ -102,7 +109,7 @@ public static void AddSwagger(this IServiceCollection services)
         public static void UseSwagger(this IApplicationBuilder applicationBuilder)
         {
             applicationBuilder.UseSwagger(c =>
-            {
+            {                
```

Comment:
Whitespaces at the end of the line are not allowed.

Automatically format documents when saving:
Install https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.PowerCommandsforVisualStudio
Go to `Tools > Options > Productivity Power Tools > PowerCommands`
Enable `Format documents on save` and `Remove and Sort Usings on save`

Reply:
Power commands was run thru